hypha/core/templates/core/navigation/primarynav-apply.html

```diff
@@ -4,30 +4,53 @@
         <ul class="nav nav--primary" role="menubar">
             {% apply_nav_items request.user as nav_items %}
             {% for item in nav_items %}
-                {% url item.url as target_url%}
-                <li class="nav__item" role="presentation">
+                <li class="nav__item"
+                    role="presentation"
+                    x-data="{open: false}"
+                    x-on:mouseover.away="open = false"
+                >
                     {% if item.sub_items %}
-                        <div class="relative inline-block" x-data="{open: false}">
-                            <a class="nav__link {% if target_url in request.path %}nav__link--active{% endif %}"
-                               href="{% url item.url %}"
+                        <div class="relative inline-block">
+                            <a class="nav__link {% if item.url in request.path %}nav__link--active{% endif %}"
+                               href="{{item.url}}"
                                aria-label="{% trans "Menu Item" %}"
                                aria-haspopup="menu"
                                aria-expanded="false"
                                role="menuitem"
+                               @mouseover="open = true"
                                @click.prevent="open = ! open">
                                 {{ item.title }}
-                                {% heroicon_outline "chevron-down" aria_hidden="true" size=16 stroke_width=3 class="inline align-baseline mr-1" %}
+                                <span :class="open ? '[&>svg]:translate-y-1': ''">
+                                    {% heroicon_outline "chevron-down" aria_hidden="true" size=16 stroke_width=3 class="inline align-baseline mr-1 transition-transform" %}
+                                </span>
                             </a>
-                            <div x-cloak x-trap="open" x-on:keydown.escape="open = false" x-show="open" x-transition @click.outside="open = false" class="absolute block bg-white min-w-max shadow-lg z-1 mt-7 ">
+                            <div
+                                x-cloak
+                                x-trap.noautofocus="open"
+                                x-on:keydown.escape="open = false"
+                                x-show="open"
+                                x-transition
+                                @click.outside="open = false"
+                                class="min-w-48 absolute block bg-white shadow-xl z-20 mt-4 border rounded-sm"
```

Comment:
`min-w-48` seems to not generate any styles? Why not use `min-w-max`? Having menu item break on two lines is undesirable.